onnx/defs/math/defs.cc

```diff
@@ -259,19 +259,17 @@ and output blobs.
                "computed element-wise");
 
 OPERATOR_SCHEMA(Pow)
-    .NumInputs(1)
+    .NumInputs(2)
     .NumOutputs(1)
-    .Attr("exponent",
-          "The exponent of the power function.",
-          AttrType::FLOAT)
-    .AllowConsumed({{0, 0}})
+    .AllowConsumed({{0, 0}, {1, 0}})
```

Comment:
probably we don't want to allow for the second argument to be consumed

Reply:
Sure